lms/djangoapps/course_home_api/outline/v1/views.py

```diff
@@ -177,6 +220,25 @@ def get(self, request, *args, **kwargs):
 
         return Response(serializer.data)
 
+    @staticmethod
+    def get_start_block(blocks, block):
+        children = blocks.get_children(block)
+        if not children:
+            return None
```

Comment:
What happens if you just return block here?

Reply:
If there happened to be no sections in a course, and we returned `block` here, it would create a link and it would just link to an empty courseware page. Wouldn't necessarily break anything, it would just be a weird experience (but I don't foresee a case where a course wouldn't have any sections...)

I also made it so that in the MFE, if there is no `resume_course_url`, we won't display the button.

Reply:
Ah okay, I wasn't sure if you used the course block, if it was smart enough to just start you off. All good!

And nice, that sounds like a good thing! Good eye

Reply:
Wait that just made me realize, regardless of whether a course has children or not, using `jump_to` with the top level course block url is smart enough to bring you back to where you left off (ex. in Demo Course http://localhost:18000/courses/course-v1:edX+DemoX+Demo_Course/jump_to/block-v1:edX+DemoX+Demo_Course+type@course+block@course will bring me to `.../block-v1:edX+DemoX+Demo_Course+type@chapter+block@interactive_demonstrations` which is the correct resume block...)

I wonder if finding the 'resume_block' is unnecessary logic? Or if it's useful for cases where we're not building the URL using `jump_to`?

I'm confused 🙃